control-plane/api-gateway/gatekeeper/gatekeeper.go

```diff
@@ -76,14 +87,14 @@ func (g *Gatekeeper) Delete(ctx context.Context, nsname types.NamespacedName) er
 // resourceMutator is passed to create or update functions to mutate Kubernetes resources.
 type resourceMutator = func() error
 
-func (g Gatekeeper) namespacedName(gateway gwv1beta1.Gateway) types.NamespacedName {
+func (g *Gatekeeper) namespacedName(gateway gwv1beta1.Gateway) types.NamespacedName {
```

Comment:
We were using pointer receivers everywhere else except for this func and the one immediately below